services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -69,15 +83,161 @@ const Asset = ({ libraryData }) => {
     }
   ]
 
+  const assetPath = breadcrumbItems[1].path
+
   const seo = {
     title: name,
     description
   }
 
+  const { sponsor } = assetData.params
+  const SponsorIcon = teams[sponsor] && teams[sponsor].icon
+
+  const AssetItemMeta = ({ properties }) => {
```

Comment:
I'm not following why this is a component? I see it's used once and it's specific to rendering asset status. Is this component needed?